icdiff

```diff
@@ -303,6 +312,9 @@ class ConsoleDiff(object):
         # markup
         fromlines, tolines = self._tab_newline_replace(fromlines, tolines)
 
+        if self.strip_trailing_cr:
+            fromlines, tolines = self._strip_trailing_cr(fromlines, tolines)
```

Comment:
I'd find this clearer as:

```
fromlines = self._strip_trailing_cr(fromlines)
tolines = self._strip_trailing_cr(tolines)
```

There's no reason why `_strip_trailing_cr` would want both lists of lines at once